code/game/mecha/working/ripley.dm

```diff
@@ -66,4 +67,51 @@
 	for(var/obj/item/mecha_parts/mecha_tracking/B in src.contents)//Deletes the beacon so it can't be found easily
 		qdel (B)
 
+// VORESTATION EDIT BEGIN
 
+/obj/mecha/working/ripley/verb/detect_ore()
+	set category = "Exosuit Interface"
+	set name = "Detect Ores"
+	set src = usr.loc
+	set popup_menu = 0
+
+	usr << "You start scanning for metal deposits."

```

Comment:
to_chat() and other stream operators below

I'm not a huge fan of the copypaste from ore scanner. If the scanner changes (like if ore categories change) you'd have to remember to paste it here. I assume this is just because you can't make do_after interrupt if they move the mech?